src/components/SignIn/SignIn.js

```diff
@@ -72,7 +82,8 @@ class SignIn extends Component {
         }       
     };   
 
-    inputChangeHandler = (event, inputIdentifier) => {        
+    inputChangeHandler = (event, inputIdentifier) => { 
+        console.log('Ia, here')       
```

Comment:
remove all console.log

Reply:
Component is not needed anymore. It has been removed.